python/tk_desktop/notifications/notification_manager.py

```diff
@@ -114,19 +111,21 @@ def get_notifications(self):
             )
             to_return = [first_launch_notif]
             for notif in other_notifs:
-                accepted = True
-                for notif_class in self.NOTIFS_TO_BE_INCLUDED_IN_FIRST_LAUNCH:
-                    if not isinstance(notif, notif_class):
-                        self.dismiss(notif)
-                        accepted = False
-                        break
-                if accepted:
+                if self._is_notification_to_be_included_in_first_launch(notif):
```

Comment:
🙌

Reply:
For the record: the previous code was not working if NOTIFS_TO_BE_INCLUDED_IN_FIRST_LAUNCH is empty